src/custom/state/enhancedTransactions/updater/FinalizeTxUpdater.tsx

```diff
@@ -69,12 +69,6 @@ function finalizeEthereumTransaction(
   const { chainId, addPopup, dispatch } = params
   const { hash } = transaction
 
-  const ethFlowInfo = transaction.ethFlow
-  if (ethFlowInfo) {
-    params.removeInFlightOrderId(ethFlowInfo.orderId)
-    setTimeout(() => params.removeInFlightOrderId(ethFlowInfo.orderId), DELAY_REMOVAL_ETH_FLOW_ORDER_ID_MILLISECONDS)
-  }
-
```

Comment:
Moved this down with the existing conditional

Also removed line `74` as it makes the timeout irrelevant
Or did I read this wrong?

@anxolin

Reply:
🙈 I think i had an issue rebasing, where I probably click on keep both. Good catch